src/frontend/komponenter/Høyremeny/Steg.ts

```diff
@@ -6,6 +6,9 @@ export enum Steg {
     BESLUTTE_VEDTAK = 'BESLUTTE_VEDTAK',
     IVERKSETT_MOT_OPPDRAG = 'IVERKSETT_MOT_OPPDRAG',
     VENTE_PÅ_STATUS_FRA_ØKONOMI = 'VENTE_PÅ_STATUS_FRA_ØKONOMI',
+    VILKÅRSVURDERE_STØNAD = 'VILKÅRSVURDERE_STØNAD',
+    REGISTRERE_OPPLYSNINGER = 'REGISTRERE_OPPLYSNINGER',
+    VILKÅRSVURDERE_INNGANGSVILKÅR = 'VILKÅRSVURDERE_INNGANGSVILKÅR',
```

Comment:
Tror ikke disse skal med. Er nok heller tekstmappingen som skal slettes. Dette er bare et steg som heter VILKÅR nå.

Reply:
Mulig `REGISTRERE_OPPLYSNINGER` fortsatt er et steg

Reply:
Her ser du hvilke steg vi skal ha: enum class StegType(val rekkefølge: https://github.com/navikt/familie-ef-sak/blob/b97bc3d4b82a7e45e4821278776f0d39763a4202/src/main/kotlin/no/nav/familie/ef/sak/service/steg/BehandlingSteg.kt#L32

Reply:
La dem inn siden klar-teksten klagde på at den ikke hadde alle enums, men okei. Da fjerner jeg de tilsvarende tekstene. Takk for lenke @eirarset